card.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	]
+}
```

Comment:
what's this?

Reply:
> what's this?

I don't understand. 
how was it created? and what is he for?